psamm_import/importer.py

```diff
@@ -124,6 +124,26 @@ def decimal_representer(dumper, data):
     return dumper.represent_scalar('tag:yaml.org,2002:float', value)
 
 
+def detect_extracellular(model):
+    """Detect the identifier for equations with extracellular compartments,
+        as it's not always the same.
+    """
+    extracellular_key = Counter()
+
+    for reaction_id, reaction in iteritems(model.reactions):
+        if 'equation' not in reaction.properties:
+            continue
+
+        equation = reaction.properties['equation']
+        if len(equation.compounds) == 1:
+            compound, _ = equation.compounds[0]
+            compartment = compound.compartment
+            extracellular_key[compartment] += 1
+    best_key, _ = extracellular_key.most_common(1)[0]
```

Comment:
Just to be safe I would add a check before `...most_common(1)[0]` to see if `len(extracellular_key) == 0`. If that is the case the function should return `None` immediately. Without this check the program will crash if the model does not have any exchange reactions.